bin/setup

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
+
 set -euo pipefail
 IFS=$'\n\t'
 
 bundle install
 
-# Do any other automated setup that you need to do here
+for dir in spec/integration/*; do
+    pushd "$dir"
```

Comment:
Is this a tab?! We have a 2 space thing everywhere ;)

Reply:
Haha, that's true. My editor does tabs for shell scripts by default.